include/phpFunctions.php

```diff
@@ -566,6 +785,39 @@ function GetRecentPost(){
 		return $result['recentpost'];
 	}
 
+	function EditPost($postID, $changes, $tagarray){
+		// echo"EditPost Called!";
+		var_dump($changes);
+		foreach($changes as $column=>$change){
+			var_dump($column, $change);
+			$result = dbQuery("
+				UPDATE posts
+				SET $column = :change
+				WHERE postID = :postID
+			", array('change'=>$change, 'postID'=>$postID))->fetch();
+		}
+		AttachTags($postID, $tagarray);
+	}
+
+	function GetPostAttributeArray($postID){
+		$result = array();
+		$type = GetPost($postID)['postType'];
+		$placeholder = dbQuery("
+			SELECT *
+			FROM typeattributes
+			INNER JOIN posttypes ON typeattributes.typeID = posttypes.typeID
+			WHERE posttypes.postType = :type
+		", array('type'=>$type))->fetch();
+		foreach($placeholder as $key=>$val){
+			if($val != NULL){
+				if(($key != 'typeID')&&($key != 'postType'))
+				$result[$key] = $placeholder[$key];
+			}
+		}
+		// var_dump($result);
+		return $result;
```

Comment:
It's not totally clear to me what this function is doing without being able to see what kind of data is in the db. That doesn't mean it's bad/wrong, just that I'm not able to give it a proper review. If you want any feedback on it, maybe you can walk me through it in person. If you're pretty confident in what you're doing, no need to discuss it.